plugins/module_utils/_api_data.py

```diff
@@ -132,13 +132,23 @@ def join_path(path):
     ),
     ('interface', 'list'): APIData(
         primary_keys=('name', ),
+        fully_understood=True,
         fields={
             'comment': KeyInfo(can_disable=True, remove_value=''),
             'exclude': KeyInfo(),
             'include': KeyInfo(),
             'name': KeyInfo(),
         },
     ),
+    ('interface', 'list', 'member'): APIData(
+        primary_keys=('interface', 'list', ),
```

Comment:
While it makes no practical difference, I would change the order:
```suggestion
        primary_keys=('list', 'interface', ),
```

Reply:
I just used alphabetical order, but as you wish :)